{{cookiecutter.repostory_name}}/bin/notify.py

```diff
@@ -0,0 +1,75 @@
+import argparse
+import os
+from datetime import datetime
+
+import requests
+
+msg_template = "*{timestamp}*\n{author}\n{changelog}"
+
+
+class SlackNotifier:
+    url = "https://slack.com/api/chat.postMessage"
+
+    def __init__(self, token, channel):
+        if not channel or not token:
+            raise Exception("Missing token or channel")
+
+        self.token = token
+        self.channel = channel
+
+    def notify(self, text):
+        print(f"Sending message to channel {self.channel}")
+        r = requests.post(
```

Comment:
as this class can be used more than 2 times per minute (not in this PR but in the future), make requests.Session in __init__ and use it here to prevent breaking a https connection after every request